docs/oauth2.md

```diff
@@ -0,0 +1,13 @@
+# Integration with OAuth2 providers
+
+Fractal supports OAuth2 login using GitHub, Google or custom providers.
+
+First, configure the server as described in [OAuth2 documentation of fractal-server](https://fractal-analytics-platform.github.io/fractal-server/internals/users/#oauth2).
+
+Fractal-web callback has to be defined using the following path: `/auth/login/oauth2`.
+
+Then, enable the desired provider on fractal-web using the `PUBLIC_OAUTH_PROVIDER` environment variable, for example:
+
+```
+PUBLIC_OAUTH_PROVIDER=github
```

Comment:
A quick clarification:

This variable is meant to match the client name in https://fractal-analytics-platform.github.io/fractal-server/internals/users/#configuration. That is, it could be e.g. "github", "google", or "myclient" or anything else that is part of the fractal-server endpoint `/auth/client-name/authorize`.

Is that correct?

Reply:
I confirm. Do you prefer that I rename it to `PUBLIC_OAUTH_CLIENT_NAME`? We could also clarify this in the docs.

Reply:
> Do you prefer that I rename it to PUBLIC_OAUTH_CLIENT_NAME? We could also clarify this in the docs.

I have no strong preference about the variable name, but +1 for making it clear in the docs.

Reply:
Done. I'm going to merge this.